### PR TITLE
Add crazyhouse test, enhance output format, and update README

### PR DESCRIPTION
**Pull request by ddugovic**
After much research and analysis I discerned usage of various parameters and introduced a multi-variable crazyhouse test.

### CODE REVIEW

File: README [hunk 1]
```diff
@@ -98,6 +98,20 @@ Notes:
...
+There are 3 main knobs, the two gain sequences (_a_ and _c_), and the perturbation distribution, delta.
+Delta is best chosen as Bernoulli (+/- 1), as that is asymptotically optimal (it must not be Normal or Uniform, see ISSO).
+There are rules about the properties of the gain sequences. The standard form works well, and follows the forms:
+
+```
+a(k) = a / (k + 1 + A) ^ alpha
+c(k) = c / (k + 1) ^ gamma
+```
+
+Here, the values alpha and gamma are tied together, asymptotically optimial is _alpha_ = 1 and _gamma_ = 1/6, but the values _alpha_ = .602 and _gamma_ = .101 work well for finite cases.
```
Comment (lakinwecker) [inline README]: `optimial` -> `optimal` ?

Comment (ddugovic) [reply README]: Ah, yes, that is a typo.

**Comment by ianfab** [comment]
Thanks. I have not worked through the changes of the readme, but since you have done much research on this you are anyhow more qualified to judge this. The simplifications and the additional output look good to me, so I will merge this.

In case you use the added example for the tuning session you mentioned elsewhere, it will be interesting to see whether the tuned values work well in tests with longer time controls, since I assume that the margins are sensitive to search depth, especially in crazyhouse.

**Comment by ddugovic** [comment]
I fully agree, having watched Spall's lecture and read about the subject, it should be interesting to test whether the theory applies in practice to Stockfish, and whether tuned parameters are resilient to LTC.

**Comment by ddugovic** [comment]
Long-term I hope that proper formatting and interpretation of data will yield more efficient techniques to tune the tuner.

**Pull request merged**